integration-tests/README.md

```diff
@@ -0,0 +1,12 @@
+# Integration Tests
```

Comment:
Good question! I put the integration tests in a separate crate so that its [build script](https://github.com/NearSocial/social-db/blob/5d751a8e3d4413451d893f88a764aead7b595f87/integration-tests/build.rs) can build the WASM contract before running the tests. We could potentially make this work if the integration tests lived in the contract crate, but it would be quite hacky as the build target for tests would be native, but the contract needs to be compiled with WASM target.

Regarding naming of the integration tests directory, I followed the pattern in `create-near-app`, but happy to adjust.

Note that there is a top level `tests` folder in the repo with a single noop test that is not part of a crate and is not a member of the workspace. Maybe we could remove that?

Reply:
I removed the tests folder now in [ae24bfe](https://github.com/NearSocial/social-db/pull/11/commits/ae24bfe95443f37d61fe8157ee6094f204de6385)